bazel/BUILD

```diff
@@ -211,6 +211,11 @@ config_setting(
     values = {"define": "log_debug_assert_in_release=enabled"},
 )
 
+config_setting(
+    name = "include_known_issue_asserts",
+    values = {"define": "include_known_issue_asserts=true"},
```

Comment:
Assuming we go with this change, I think this should be inverted. The default should still be for assertions to be enabled. If someone is hitting an issue and can't debug/fix they can disable the assertion.

Reply:
Sure, I'm fine with inverting it. Will update.